Sources/XCUITestHelpers/XCUIDevice+UserInterfaceStyle.swift

```diff
@@ -0,0 +1,32 @@
+import UIKit
+import XCTest
+
+public extension XCUIDevice {
+
+    /// The device `UIUserInterfaceStyle`.
+    ///
+    /// This is declared as `static` rather than at the instance level to make it easier to call.
+    /// The UI tests thread can't run on multiple devices at the same time, so there is no strict
+    /// need to read this value from an instance.
```

Comment:
Now that we are officially working on this library, I feel strongly every `public` method should be documented as best as we can.

We want to make it as easy as possible for developers to write UI tests. Adding documentation doesn't make up for software that is hard to use, or for the quirks of running UI tests but is nevertheless something we can do to lower the barrier to entry.

Reply:
I agree with this